website/docusaurus.config.js

```diff
@@ -210,14 +210,14 @@ const config = {
 									</p>
 								  `
               },
-              {
-                html: `
-									<a class='footerButton' href='https://github.com/ecadlabs/taqueria'>
-										<img class='footerGihubLogoButton' src="/img/githubSVG.svg" alt="" />
-										GITHUB
-									</a>
-								  `
-              },
+              // {
```

Comment:
Should this be removed as opposed to commented out?

Reply:
Yes, I think this is finalized now so I'll remove